packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.vitest.tsx

```diff
@@ -59,47 +54,21 @@ const renderComponent = (
   return render(getComponent(props), { wrapper: createWrapper(synapseContext) })
 }
 
-function mockGetEntityWiki(markdown: string) {
-  server.use(
-    rest.get(
-      `${getEndpoint(
-        BackendDestinationEnum.REPO_ENDPOINT,
-      )}/repo/v1/entity/:id/wiki/:wikiId`,
-      async (req, res, ctx) => {
-        const response: WikiPage = {
-          ...mockEntityWikiPage,
-          markdown,
-        }
-        return res(ctx.status(200), ctx.json(response))
-      },
-    ),
-  )
-}
+const spyOnMath = vi.spyOn(MarkdownSynapse.prototype, 'processMath')
 
-describe('MarkdownSynapse tests', () => {
-  beforeAll(() => server.listen())
-  afterEach(() => server.restoreHandlers())
-  afterAll(() => server.close())
```

Comment:
Dropped mock service worker in this particular test. In this case, it seemed to increase the complexity of mocks more than we benefit from behavior resembling a real server (though I don't know if I would made the same argument for certain other tests).